### ISSUE TITLE
The icon is missing

### ISSUE THREAD

**Issue by MaxAlekseyev** [brabitom/komodo-sites#4]
Hello,

The icon is missing in the pane tab, how I can apply or restore it?
Win7x64
Komodo IDE, version 10.1.1, build 89474, platform win32-x86.

Regards, Max

**Comment by brabitom** [comment]
Hi, it's global problem with Komodo 10 (regardless of the OS). 
I tried to fix it, but without success so far.

**Comment by MaxAlekseyev** [comment]
Is that reasonable to ask the main forum if the problem is the complex or this problem well known in development community and there is now solution yet for actual versions?

**Comment by brabitom** [comment]
Yes, you're right, but I don't have time to solve it now.
If you have time, you can try to fix it and pull request with solution.

**Comment by MaxAlekseyev** [comment]
Yes, I understand that this task is not in highest priority. I'll try to check if I can do something if I have time.
Can you tell me the path of the installed module location. There very strange names assigned for some modules.
